test/e2e/form-validation.test.js

```diff
@@ -1,14 +1,126 @@
 const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlFormValidationPage = require('./pages/vl-form-validation.page');
+const { Key } = require('selenium-webdriver');
 
 describe('vl-form-validation', async () => {
-   const vlFormValidationPage = new VlFormValidationPage(driver);
+    const vlFormValidationPage = new VlFormValidationPage(driver);
+    
+    beforeEach((done) => {
+        vlFormValidationPage.load().then(() => {
+            done();
+        });
+    });
+    
+    it('Als gebruiker zie ik dat alle verplichte inputvelden in het eerste en tweede formulier correct gekoppeld zijn met de corresponderende validation-message', async() => {
+        var forms = [1, 2];
```

Comment:
nuttig da ge 2 keer hetzelfde test? enige verschil tussen da 1e en 2e is toch die extra succes-melding? dan komt ge weer bij de discussie van vorige week uit denk ik ...

Reply:
ik zie hier ook het nut van de test niet helemaal in. zolang ge niks 'invalid' invult in de velden en er dus geen foutboodschap verschijnt, kunt ge toch testen dat attribuutjes aan mekaar gelijk zijn zoveel ge wilt? als dat uiteindelijk niks blijkt te doen ...

Reply:
Mijn bedoeling was om hier te testen dat de 'setup' om de foutmeldingen te tonen correct was, maar ik wil dit gerust wegdoen.